### PR TITLE
Add share buttons for social medias

### PR DESCRIPTION
**Pull request by uilianries**
- Add support for sharing any blog post using social medias: Twitter, Linkedin, Facebook, ~GooglePlus~ and e-mail 
- I've followed Jekyll docs: https://jekyllcodex.org/without-plugin/share-buttons/
- Google just ended G+, so I've removed it from the list.
- We could add more, but not sure, as reddit
- There is no comment when sharing, only the URL. We can customize of course.
- I didn't try with Facebook, because, well, I haven't used it in years.

Preview:

**Buttons**
Screenshot_2019-09-27 Package ID modes control the ABI and the traceability of your dependencies(2)

**Twitter sharing**
Screenshot_2019-09-27 Post a Tweet on Twitter

**Linkedin sharing**
Screenshot_2019-09-27 (13) LinkedIn

### CODE REVIEW

**Comment by danimtb** [comment]
Cool! Can we also include the title of the post with the URL as well?

File: _includes/share-buttons.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    float: left;
+    text-align: center;
+}
+#share-buttons > div > svg {height: 16px; fill: #d5d5d5; margin-top: 10px;}
+#share-buttons > div:hover {cursor: pointer;}
+#share-buttons > div.facebook:hover > svg {fill: #3B5998;}
+#share-buttons > div.twitter:hover > svg {fill: #55ACEE;}
+#share-buttons > div.linkedin:hover > svg {fill: #0077b5;}
+#share-buttons > div.pinterest:hover > svg {fill: #CB2027;}
+#share-buttons > div.gplus:hover > svg {fill: #dd4b39;}
```
Comment (danimtb) [inline _includes/share-buttons.html]: Can we remove this? I'd like to include dead code...

Comment (danimtb) [reply _includes/share-buttons.html]: Those like instragram, pinterest, gplus are not needed

**Comment by memsharded** [comment]
This is great, many thanks!

Could we have the title of the post in the Twit too? Not only the link. Or is it too complicated?

**Comment by danimtb** [comment]
@uilianries have a look at https://github.com/uilianries/conan-io.github.io/pull/1

**Comment by uilianries** [comment]
> This is great, many thanks!
> 
> Could we have the title of the post in the Twit too? Not only the link. Or is it too complicated?

@memsharded Yes, @danimtb included the title at uilianries#1

**Comment by danimtb** [comment]
@uilianries could you please verify the buttons are working with the title as expected?

**Comment by uilianries** [comment]
Facebook won't share the text, due a limitation from their side: https://stackoverflow.com/questions/5023602/facebook-share-link-can-you-customize-the-message-body-text

I gonna post the preview of each one ...

**Comment by uilianries** [comment]
Blog post preview:

Screenshot_2019-09-30 Package ID modes control the ABI and the traceability of your dependencies


GMail composition:

Screenshot_2019-09-30 Compose Mail - uilianries gmail com - Gmail

Linkedin:

Screenshot_2019-09-30 (11) LinkedIn


Twitter (it includes the title):

Screenshot_2019-09-30 Post a Tweet on Twitter


Look mom, I have facebook!

Screenshot_2019-09-30 Publicar no Facebook

**Comment by uilianries** [comment]
@danimtb I think we are good to go!

**Comment by memsharded** [comment]
Great job!

**Pull request merged**